Packs/Azure-Enrichment-Remediation/Playbooks/Azure_-_Enrichment.yml

```
@@ -2,750 +2,952 @@ id: Azure - Enrichment
+          right:
+            value: {}
+      label: "yes"
+    continueonerrortype: ""
     id: "14"
-    taskid: 145b8864-a6c4-46aa-8e65-2c9f3efba993
-    type: condition
-    task:
-      id: 145b8864-a6c4-46aa-8e65-2c9f3efba993
-      version: -1
-      name: Is Microsoft Graph User enabled?
-      description: Determines if the Microsoft Graph User integration instance is configured.
-      type: condition
-      iscommand: false
-      brand: ""
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
     nexttasks:
       '#default#':
       - "7"
       "yes":
       - "13"
+    note: false
+    quietmode: 0
     separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if the Microsoft Graph User integration instance is configured.
+      id: d689060f-4d97-40db-8332-b6d382f29ec8
+      iscommand: false
+      name: Is Microsoft Graph User enabled?
+      type: condition
+      version: -1
+    taskid: d689060f-4d97-40db-8332-b6d382f29ec8
+    timertriggers: []
+    type: condition
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 2580
+        }
+      }
+  "15":
+    continueonerrortype: ""
+    id: "15"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "16"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      query:
+        complex:
+          root: inputs.AzureIP
+          transformers:
+          - args:
+              prefix:
+                value:
+                  simple: resources | where type == 'microsoft.network/publicipaddresses'| where properties['ipAddress'] == '
+              suffix:
+                value:
+                  simple: ''' | project name, id, tenantId, subscriptionId, resourceGroup | join kind=inner (resourcecontainers | where type == ''microsoft.resources/subscriptions'' | project subscriptionId, subscriptionName=name, properties.managementGroupAncestorsChain) on subscriptionId'
+            operator: concat
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: Azure Resource Graph
+      description: Executes a given Azure Resource Graph Query to find management group information.
+      id: 0b468465-df26-4641-8b01-cfee8a4a3a23
+      iscommand: true
+      name: Query for management group chain
+      script: Azure Resource Graph|||azure-rg-query
+      type: regular
+      version: -1
+    taskid: 0b468465-df26-4641-8b01-cfee8a4a3a23
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1290
+        }
+      }
+  "16":
+    continueonerrortype: ""
+    id: "16"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "8"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      key:
+        simple: azureAssetHierarchy
```

```suggestion
        simple: AzureAssetHierarchy
```
NIT make this camel case as all others are